runtime/v2/sdk/src/main/java/com/walmartlabs/concord/runtime/v2/sdk/Execution.java

```diff
@@ -44,6 +44,8 @@
     @Nullable
     Step currentStep();
 
+    String currentFlowName();
```

Comment:
This TODO can be removed
https://github.com/walmartlabs/concord/blob/233a2967b0a500dbb4c166e5f2584d3001ce22e1/runtime/v2/sdk/src/main/java/com/walmartlabs/concord/runtime/v2/sdk/Execution.java#L34

Reply:
:)